recipes-core/lightdm-fb/files/postinst

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+# Add support for hotplug of Display Port
+sed -i 's/LABEL="systemd_end"/SUBSYSTEM=="graphics", KERNEL=="fb0", SYMLINK="fb0", TAG+="systemd"\nLABEL="systemd_end"/g' /lib/udev/rules.d/99-systemd.rules
+
+sed -i '/plymouth/d' /lib/systemd/system/lightdm.service
```

Comment:
Nope, thats's what `/etc/systemd` is for.

Reply:
it is also unclear what we are doing here (I mean I understand that we are essentially killing every single line where the word plymouth can be found but why?)